internal/appconfig/config.go

```diff
@@ -147,3 +147,10 @@ func (c *Config) DockerBuildTarget() string {
 	}
 	return c.Build.DockerBuildTarget
 }
+
+func (c Config) GetInternalPort() int {
```

Comment:
Getter doesn't "Get" it :) please see https://go.dev/doc/effective_go#Getters

Reply:
I only prefixed it with "Get" because I wanted to have one function that validates and safely extracts the port.
what would you suggest i name it, cos i really cant think of a better name.

Reply:
just `InternalPort()` or `ServiceInternalPort` if you prefer.